app/models/assignment.rb

```diff
@@ -756,6 +756,55 @@ def groups_submitted
     groupings.select(&:has_submission?)
   end
 
+  def get_num_assigned(assignment, ta_id = nil)
```

Comment:
This is better, but one thing I forgot to mention last time is that these are all instance methods, so passing in an extra `assignment` argument is pretty redundant.

Basically just remove all references to `assignment` in these methods. It will still work the same way.